src/docs/guides/contributing/contributing-code.md

```diff
@@ -0,0 +1,72 @@
+- If you want to start a conversation with other community members or run the CI workflows but you're not done yet, open your PR as a [draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests). Then, [change it to ready for review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
+- If your PR addresses an issue, be sure to [link them](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). This helps everyone find their way around contributions, and merging your PR will automatically close the issue too.
+- If there's no issue for your PR, then please describe the context and rationale in the pull request description, and provide screenshots/screen recordings of the changes if they affect the UX.
+- Refactoring is great, but if you do so, please guard it with new tests.
+- If you add a significant new feature or a breaking change, then document this under the release notes of the upcoming pull request.
+- So we can help you better, please [allow our core contributors to edit your PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
+
+!!! tip
+    When in doubt, just ask the community for guidance, we're happy to help!
+
+## Feedback
+
+All code submissions will be reviewed and tested by the core team, and only those that meet our bar for both quality and design/roadmap appropriateness will be merged. Please be patient; we very much appreciate you contributing to Orchard Core!
+
+!!! note
+    Following the below recommendations will ensure that your PR is merged as quickly as possible. Please work with us so it's a pleasant experience for everyone.
+
+- Please update your pull request according to feedback until it is approved by one of the core team members.
+- Apply [suggested changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) directly so the reviewer doesn't have to eyeball the changes. These resolve themselves after applying them, and that's fine.
+- Don't resolve other conversations so it's easier to track for the reviewer. Then, the reviewer will resolve them.
+- Feel free to mark conversations that you addressed to keep track of them with an emoji or otherwise, just don't resolve them.
+- Please keep conversations happening in line comments in those convos, otherwise, communication will be a mess. If you have trouble finding them, see [this video](https://github.com/OrchardCMS/OrchardCore/pull/14749#issuecomment-1917976028).
```

Comment:
```suggestion
- Please keep conversations happening inline comments in those convos, otherwise, communication will be a mess. If you have trouble finding them, see [this video](https://github.com/OrchardCMS/OrchardCore/pull/14749#issuecomment-1917976028).
```

Reply:
I mean "line comments" here, i.e. comments added on lines under the PR, not "inline comments".